docker-compose.yml

```diff
@@ -27,22 +27,35 @@ services:
     volumes:
       - ./back/src:/svr/app/src:ro
       - ./back/test:/svr/app/test:ro
+      - ./back/prisma:/svr/app/prisma:ro
```

Comment:
[dust] I believe it's not necessary, I'll test tomorrow on executing some requests

Reply:
It seem to be finally